airflowHPC/dags/replex.py

```diff
@@ -225,7 +225,7 @@ def calc_prob_acc(swap, dhdl_files, states):
     from alchemlyb.parsing.gmx import _extract_dataframe as extract_dataframe
 
     logging.info(dhdl_files)
-    shifts = [SHIFT_RANGE for i in range(NUM_SIMULATIONS)]
+    shifts = [SHIFT_RANGE] * NUM_SIMULATIONS
```

Comment:
@wehs7661 FYI you can do this lazy list initialization if you have a list of repeating value.